endorsement/storeapi.proto

```diff
@@ -43,6 +43,76 @@ message OpenArgs {
 
 message CloseArgs {
 }
+message SwComponentID {
+    common.TokenFormat type = 1;
+    google.protobuf.Struct id = 2;
+}
+// SwComponentInfo defines the structure of a single Software Component 
+// identified by a SoftwareComponent Identifer.
+message SwComponentInfo {
+    SwComponentID comp_id = 1;
+    google.protobuf.Struct comp_meta = 2; 
+}
+
+message AddSwComponentArgs {
+    repeated SwComponentInfo info = 1;
+}
+
+message GetSwComponentArgs {
+    string software_comp_id = 1;
```

Comment:
we can simply call this `id`

Reply:
Yes, I noticed it already, will correct it!